.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "src/3rdparty/libcrashreporter-qt"]
 	path = src/3rdparty/libcrashreporter-qt
 	url = git://github.com/dschmidt/libcrashreporter-qt.git
+[submodule "src/3rdparty/qwamp"]
+	path = src/3rdparty/qwamp
+	url = git@github.com:unitednetworks/qwamp.git
+[submodule "src/3rdparty/qmsgpack"]
+	path = src/3rdparty/qmsgpack
+	url = git@github.com:romixlab/qmsgpack.git
```

Comment:
Heads-up to @hodyroff regarding licenses.

Reply:
https://github.com/unitednetworks/qwamp
says Apache 2.0 correct? Thats perfect and I'll add it to the list.
https://github.com/romixlab/qmsgpack/blob/master/LICENSE
is MIT right? Also perfect ... I do need to add both, right?

Reply:
yes both